src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  background-color: $standart-color;
+  border: 1px solid $card-border-color;
+  border-radius: $main-border-radius;
+
+  padding: 0 16px 16px;
+
+  &-container {

```

Comment:
what is this? BEM? 
![image](https://user-images.githubusercontent.com/99331793/200279245-74e2f0e8-81f7-4ae2-8232-4eb393b5e1d4.png)
why does it located outside of block?